src/main/scala/org/broadinstitute/dsde/firecloud/service/NIHService.scala

```diff
@@ -48,19 +48,15 @@ trait NIHService extends HttpService with PerRequestCreator with FireCloudDirect
                 val lastLinkTime = DateUtils.now
                 val linkExpireTime = DateUtils.nowPlus30Days
 
-                // update this user's dbGaP access in rawls, assuming the user exists in the whitelist.
-                // this call is silent/async - if it fails, we continue on, and the user's access will
-                // be updated the next time sync_whitelist runs; the user will have to wait for that.
-                val groupUpdate = perRequest(requestContext, Props(new ProfileClientActor(requestContext)),
-                  ProfileClient.SyncSelf(userInfo))
-
```

Comment:
In the original code, this is one of two PerRequests are fired off. Whichever one finishes first is the one that the client receives.